src/Sulu/Component/Rest/ListBuilder/Metadata/VirtualPropertyMetadata.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Metadata;
+
+/**
+ * Container for virtual-property-metadata.
```

Comment:
What is `VirtualPropertyMetadata`? Would be better part of the comment. Or is that already described somewhere else?